skactiveml/pool/_core_set.py

```diff
@@ -0,0 +1,309 @@
+"""
+Module implementing various Core-set Selection strategies.
+
+Core-set selection problem aims to find a small subset given a large labeled
+dataset such that a model learned over the small subset is competitive over the
+whole dataset.
+"""
+
+import numpy as np
+
+from ..base import SingleAnnotatorPoolQueryStrategy
+from ..utils import MISSING_LABEL, labeled_indices, unlabeled_indices
+from sklearn.utils.validation import check_array, check_consistent_length, check_random_state, column_or_1d
+from sklearn.metrics import pairwise_distances
+
+
+class CoreSet(SingleAnnotatorPoolQueryStrategy):
+    """Core Set Selection
+
+    This class implement various core-set based query strategies, i.e., the
+    standard greedy algorithm for k-center problem [1], the robust k-center
+    algorithm [1].
+
+    Parameters
+    ----------
+    method: {'greedy', 'robust'}, default='greedy'
+        The method to solve the k-center problem, k-center-greedy and robust
+        k-center are possible. So far only `method=greedy` is supported
```

Comment:
@mherde Remove for now this parameter will be added, once the robust method is implemented.